documentation/src/docs/deployment/run-local.md

```diff
@@ -89,29 +96,33 @@ Configure standalone server to communicate using SSL/TLS. To do so, edit the TLS
   singlenode.trustStoreJKS=../config/standalone.truststore.jks
 
   ```
-These properties include ports for Zookeeper, Segment Store and Controller. They also contain other configurations related to security.
 
-## Running standalone with encryption and authentication enabled
+2. Ensure that the server's certificate is trusted. If you run `./gradlew startStandalone` without it, you'll encounter the following error:
+
+  ```java
+  Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
```

Comment:
Please remove reference to this exception, as suggested earlier.

Reply:
@ravisharda I am not clear with your comment.! What `references` are made?? not clear...u mean the blue font....if so (that is rendered for java format)!

Reply:
What I meant was we should remove the exception from the documentation. Please remove the following: 

```
If you run `./gradlew startStandalone` without it, you'll encounter the following error:

 Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
...
```